headers/Entity.h

```diff
@@ -18,12 +18,15 @@ class Entity {
     EntityType type;
 
 public:
-    Entity(EntityType, float, float);
+    Entity(EntityType, float, float, float, float);
     Entity(EntityType, float, float, std::unordered_map<std::string, Animation>&);
     const sf::Vector2f &getTransform() const;
     void Update(const sf::Event &);
     void Draw(sf::RenderTarget &);
     void Reset();
+    sf::Vector2f GetTransform() const; 
```

Comment:
Looks like this is a duplicate of `getTransform()` :)

Probably the `const sf::Vector2f &() const` signature is better, but for consistency this capitalisation is better.